src/main/java/org/ballerinax/datamapper/DataMapperPlugin.java

```diff
@@ -579,4 +589,19 @@ private String getCustomizedErrorMessage(Exception e) {
 
         return errorMessage;
     }
+
+    private static boolean hasAcceptedCharacters(String input) {
+        return input != null && input.matches("^[a-zA-Z0-9_:/.]*$");
```

Comment:
Shall we use a compiled regex here. https://docs.oracle.com/javase/7/docs/api/java/util/regex/Pattern.html

Reply:
With the latest improvements made to the code we will remove this method `hasAcceptedCharacters(String input)`